### PR TITLE
Add rpm-tests support for external contributors

### PR DESCRIPTION
**Pull request by jkonecny12**
After this is merged, owners can start ``rpm-tests`` with a ``unit-tests`` by comment ``/test`` comment in Pull request.

Tested [here](https://github.com/Test-anaconda-org/anaconda/pull/9).

### CODE REVIEW

File: .github/workflows/validate-rhel-8.yml [hunk 1]
```diff
@@ -97,3 +99,58 @@ jobs:
...
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ needs.pr-info.outputs.sha }}
+          fetch-depth: 0
+
+      - name: Rebase to current master
```
Comment (VladimirSlavik) [inline .github/workflows/validate-rhel-8.yml:127]: Not master here, I guess...

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

File: .github/workflows/validate-rhel-8.yml (hunk 1 above)
Comment (jkonecny12) [reply .github/workflows/validate-rhel-8.yml]: Fixed, thanks!

**Comment by jkonecny12** [comment]
[Run](https://github.com/Test-anaconda-org/anaconda/runs/1663254866?check_suite_focus=true) after my last update.

**Review by VladimirSlavik — APPROVED** [review]

**Pull request merged**